reverse_script.py

```diff
@@ -0,0 +1,64 @@
+import os, sys
+import re
+import json
+
+# DESCRIPTION
+# We are launching script from the command line 
+# Taking the name of the file with the dictionary (json) from the command
+# Reversing dictionary
+# Writing the result to the file (the name of the file present in system variable)
+
+# reverting dictionary (changing keys with values)
+def reversing_dict(dictionary):
+    result_dict = {}
+    buffer = []
+    for key, value in dictionary.items():
+        if value not in result_dict.keys(): #adding new entry if it is a new key
+            result_dict[value] = key
+        else:   #adding new value to the existed one to the same key
+            buffer = list(result_dict[value])
+            buffer.append(key)
+            result_dict[value] = buffer
+    return result_dict
+
+# set up a path to the reseult file as constanta 

```

Comment:
Usually, it is using 'if __name__ == "__main__"' construction for main part of script
https://stackoverflow.com/questions/419163/what-does-if-name-main-do

Reply:
fixed in f6c1ecd3e51d763fcd7446c3ac8abefabefafea2